CHANGELOG.md

```diff
@@ -19,6 +19,8 @@ and since Bareos version 20 this project adheres to [Semantic Versioning](https:
 - Bareos 22 removes bareos-webui support for RHEL 7 and CentOS 7
 
 ### Added
+- webui links update [PR #1275]
```

Comment:
This line belongs into the Changed section (or even Fixed).
It would be good if you could add some more information, like:
`
webui: adapt links to new URLs after website relaunch.
`
The commit message could also contain the additional hint:
`
The old URLs still work, thanks to redirection. Anyhow, it makes sense to directly use the new URLs.
`

Reply:
Thank you for the propper information. I'll fix the entry and expand the messages.